### ISSUE TITLE
Filter diseases by species

### ISSUE THREAD

**Issue by monicacecilia** [monarch-initiative/monarch-ui#29]
It would be very useful to show the diseases according to species on the landing page following a search on say "Marfan syndrome."

--------
Originally submitted by @pnrobinson to /monarch-app repo.

**Comment by kshefchek** [comment]
Can we flesh out the requirements for this?  Are we talking the breakdown of species that are models of Marfan syndrome?  ie some view of the data when you click the filter button here - https://monarchinitiative.org/disease/MONDO%3A0007947#models

**Comment by pnrobinson** [comment]
The page you are refering to is good (although the sorting is not perfect in that human and mouse are mixed). The problem page in my opinion is the landing page after a search
https://monarchinitiative.org/search/marfan
For both pages, it might be better to return the results like
human 20
mouse 32
zebrafish 3
etc
and then go to the details -- (question for Julie). At least for me there is too much information here and it might be nicer to return the information in a structured way.

**Comment by jmcmurry** [comment]
The problem can't be solved on the front end. First and foremost, MONDO [must distinguish human and animal diseases](https://github.com/monarch-initiative/mondo/issues/190).

**Comment by cmungall** [comment]
You can tell what is a non-human disease. This is sufficient for a useful filter.

**Comment by jmcmurry** [comment]
@cmungall I'm not sure what you're suggesting. Are you suggesting that we on the front-end filter out all OMIA as non-human, or something else?

**Comment by cmungall** [comment]
The conservative thing is to exclude anything with in_taxon != NCBITaxon:9606 (and allow the user to see these by ticking a "show non-human animal diseases" box

**Comment by jmcmurry** [comment]
Agreed, that would be fine, but it is my understanding that this taxon distinction is not in the source data to begin with, hence this ticket whose origins reach back to 2015. https://github.com/monarch-initiative/monarch-disease-ontology-RETIRED/issues/59. cc: @TomConlin ?

**Comment by cmungall** [comment]
What is lacking in MONDO is ontological commitment for likely human-only diseases, but anything that is definitively non-human should have an axiom to this effect, e.g.

http://www.ontobee.org/ontology/MONDO?iri=http://purl.obolibrary.org/obo/MONDO_0005748

**Comment by pnrobinson** [comment]
@cmungall @monicacecilia I do not understand the above discussion. BL API returns the species right? and so why can't we implement my original suggestion:
```
it might be better to return the results like
human 20
mouse 32
zebrafish 3
etc
and then go to the details -- 
```
